mw-webapp/src/component/accordion/AccordionTrigger/AccordionTrigger.module.scss

```diff
@@ -16,18 +16,24 @@
   flex: 1;
   align-items: center;
   justify-content: space-between;
-  padding: 1% 4%;
   border: none;
   background-color: transparent;
+  padding: 2%;
```

Comment:
why 2%? Should we change it? :)

Reply:
its better for design. Returned old variant

Reply:
old variant is not better than this one) why should we use percentage for the paddings, we already have common variables for such purposes

Reply:
fixed